sdk/iot/inc/az_iot_hub_properties.h

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <az_span.h>
+#include <az_result.h>
+
+#ifndef _az_IOT_HUB_PROPERTIES_H
+#define _az_IOT_HUB_PROPERTIES_H

```

Comment:
>_az_ [](start = 8, length = 4)

This need to be capital in all the files

Reply:
The C language spec states that undescore uppercase symbols are reserved for the language itself and so, to not conflict with this, we use lowercase for az